src/main/java/sg/ncl/MainController.java

```diff
@@ -3494,6 +3486,514 @@ public String adminEnergy(Model model,
+            throw new WebServiceRuntimeException(e.getMessage());
+        }
+
+        return "redirect:/admin/monthly/" + id + "/usage";
+    }
+
+    @GetMapping("/admin/statistics")
+    public String adminUsageStatistics(HttpSession session, Model model) {
+        if (!validateIfAdmin(session)) {
+            return NO_PERMISSION_PAGE;
+        }
+
+        if (!model.containsAttribute("query")) {
+            model.addAttribute("query", new ProjectUsageQuery());
+            model.addAttribute("newProjects", new ArrayList<ProjectDetails>());
+            model.addAttribute("activeProjects", new ArrayList<ProjectDetails>());
+            model.addAttribute("inactiveProjects", new ArrayList<ProjectDetails>());
+            model.addAttribute("utilization", new HashMap<String, MonthlyUtilization>());
+            model.addAttribute("statsCategory", new HashMap<String, Integer>());
+            model.addAttribute("statsAcademic", new HashMap<String, Integer>());
+        }
+
+        return "admin_usage_statistics";
+    }
+
+    @PostMapping("/admin/statistics")
+    public String adminUsageStatisticsQuery(@Valid @ModelAttribute("query") ProjectUsageQuery query, BindingResult result,
+                                            RedirectAttributes attributes, HttpSession session) {
+        if (!validateIfAdmin(session)) {
+            return NO_PERMISSION_PAGE;
+        }
+
+        List<ProjectDetails> newProjects = new ArrayList<>();
+        List<ProjectDetails> activeProjects = new ArrayList<>();
+        List<ProjectDetails> inactiveProjects = new ArrayList<>();
+        List<String> months = new ArrayList<>();
+        Map<String, MonthlyUtilization> utilizationMap = new HashMap<>();
+        Map<String, Integer> statsCategoryMap = new HashMap<>();
+        Map<String, Integer> statsAcademicMap = new HashMap<>();
+        int totalCategoryUsage = 0;
+        int totalAcademicUsage = 0;
+
+        if (result.hasErrors()) {
+            StringBuilder message = new StringBuilder();
+            message.append("Error(s):");
+            message.append("<ul class=\"fa-ul\">");
+            for (ObjectError objectError : result.getAllErrors()) {
+                FieldError fieldError = (FieldError) objectError;
+                message.append("<li><i class=\"fa fa-exclamation-circle\"></i> ");
+                message.append(fieldError.getField());
+                message.append(" ");
+                message.append(fieldError.getDefaultMessage());
+                message.append("</li>");
+            }
+            message.append("</ul>");
+            attributes.addFlashAttribute(MESSAGE, message.toString());
+        } else {
+            DateTimeFormatter formatter = new DateTimeFormatterBuilder().appendPattern("MMM-yyyy").toFormatter();
+            YearMonth m_s = YearMonth.parse(query.getStart(), formatter);
+            YearMonth m_e = YearMonth.parse(query.getEnd(), formatter);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)